src/FunctionApps/python/GenerateCSVs/fhir.py

```diff
@@ -59,7 +59,7 @@ def read_bundles_by_type(
 
 def get_container_client(url: str):
     """Use whatever creds Azure can find to authenticate with the storage container"""
-    creds = DefaultAzureCredential()
+    creds = DefaultAzureCredential(exclude_shared_token_cache_credential=True)
```

Comment:
What does this do?

Reply:
So this solves a problem I was having over the weekend trying to get azure to work. I tried to `az login`, which succeeded, but then when I ran the `generate_csvs.py` script locally, I got azure authentication errors. It was like a 7 hour debugging process, but the long and short of it is that somehow, the credential I used to log in as a superuser had been cached somewhere (perhaps the browser, perhaps Azure itself, the github issue logs for the problem weren't well documented). So, rather than fetch new ones, `az login` was just trying to reuse the cached version. But of course those creds expire after 6 hours so the process was automatically failing. This change just amounts to forcing Azure to skip one thing in its quest to find working creds. I.e. we know that the default creds are the last thing checked (environment variables first, then azure stuff, then shared cache, etc.), so this skips the thing above it, which happens to be shared cache. It doesn't impact the code at all, but it lets us circumvent any caching that may or may not have happened with login creds. I don't know if anyone else was having this issue (because who else is running a local script with az login for that many hours), but it could've happened as easily as me clicking "yes" when Azure asks if you want it to not ask you for MFA every time. It was so dumb.

Reply:
Interesting. Okay